airflow/bin/cli.py

```diff
@@ -1376,8 +1376,10 @@ def users_list(args):
 def users_create(args):
     appbuilder = cached_appbuilder()
     role = appbuilder.sm.find_role(args.role)
+    valid_roles = appbuilder.sm.get_all_roles()
```

Comment:
Can you move this inside `if` statement where it is used?

Reply:
Done